operations/package_repository_management/Nexus_Export_and_Restore.md

```diff
@@ -0,0 +1,65 @@
+# Nexus Export and Restore
+
+The current process for ensuring the safety of the nexus-data PVC is a one time, space intensive, manual process and is only recommended to be done while Nexus is in a known good state.
+A export is recommended to be done before an upgrade to enable the ability of rollback. Taking an export can also be used to improve Nexus resiliency by allowing easy fixes for data corruption.
+This process is still a run at your own risk procedure, and is not recommended for all cases.
+
+## Export
+
+Note about taking an export: There is only one export that should be taken, each time the script is run it will overwrite the old export.
+
+Prior to making an export check the size of the exported tar file (on the cluster, e.g. 3 times size of just the export), and the amount of storage the cluster has left, run the following command on a master node:
+
+```bash
+kubectl exec -n nexus deploy/nexus -c nexus -- df -P /nexus-data | grep '/nexus-data' | awk '{print "Amount of space the Nexus export will take up on cluster: "(($3 * 3)/1048576)" GiB";}' && ceph df | grep 'zone1.rgw.buckets.data' | awk '{ print "Currently used: " $7 $8 ", Max Available " $10 $11;}'
+```
+
+This command prints out the amount of space the Nexus export will take on the cluster. The command also prints the amount of space currently used in the Ceph pool that the export will be stored.
```

Comment:
The above command will return the following information:

* The amount of space the Nexus export will take on the cluster.
* The amount of space currently used in the Ceph pool that the export will be stored.
* The max amount of space available in that Ceph pool.
   If the size of the Nexus export plus the size of the currently used space is larger than the max available size, submit a help request to figure out a solution.